pkg/controller/provider/web/vsphere/host.go

```diff
@@ -115,6 +115,7 @@ type Host struct {
 	InMaintenanceMode bool          `json:"inMaintenance"`
 	ProductName       string        `json:"productName"`
 	ProductVersion    string        `json:"productVersion"`
+	Thumbprint        string        `json:thumbprint`
```

Comment:
The tag is malformed.  The Go tag must be name:"" where the body is in quotes.
Should be:
```
`json:"thumbprint"`
```